lib/policies.py

```diff
@@ -18,10 +19,14 @@ class ClusterAwareLoadBalancer:
     hostPortMap_public = {}
     unreachableHosts = []
     GET_SERVERS_QUERY = "select * from yb_servers()"
+    DEFAULT_REFRESH_INTERVAL = 300
+    refreshListSeconds = DEFAULT_REFRESH_INTERVAL
+
 
-    def getInstance():
+    def getInstance(refreshInterval):
         if ClusterAwareLoadBalancer.instance == None:
             ClusterAwareLoadBalancer.instance = ClusterAwareLoadBalancer()
+            ClusterAwareLoadBalancer.instance.refreshListSeconds = refreshInterval if refreshInterval > 0 else 300 
```

Comment:
In other drivers, we are setting it to the default of 300 seconds if the provided value is either outside the range [0 600] or invalid. It should have the same handling here as well.